adonis-typings/rabbit.ts

```diff
@@ -192,7 +192,7 @@ declare module '@ioc:Adonis/Addons/Rabbit' {
      */
     port?: number
 
-    vhost?: string
+    vhost: string
```

Comment:
Could you make it optional? Otherwise it would be a breaking change, people using older versions of the package would get type errors and need to update their codes.

Could you also add a comment to it sot it matches the other properties? Just a generic comment, like:

```ts
/**
 * The vhost
 */
```